pkg/cmd/provisioning/ibmcloud/service_id.go

```diff
@@ -213,18 +227,34 @@ func (s *ServiceID) createPolicy(policy *credreqv1.AccessPolicy) error {
 	return nil
 }
 
-func (s *ServiceID) decode() (*credreqv1.IBMCloudProviderSpec, error) {
-	// Decode IBMCloudProviderSpec
+func (s *ServiceID) extractPolicies() (policies []credreqv1.AccessPolicy, err error) {
```

Comment:
When doing this style of named return values, I prefer to not name the error `err`. Can we call it `returnErr` (or really anything more descriptive than the generic `err` name used inside the fuction)?

Reply:
fixed it. ptal latest code